### ISSUE TITLE
pip install error 'egg_base' option: 'pip-egg-info' does not exist or is not a directory

### ISSUE THREAD

**Issue by lowend** [Miserlou/SoundScrape#48]
pip install soundscrape
Downloading/unpacking soundscrape
  Downloading soundscrape-0.23.4.tar.gz
  Running setup.py (path:/tmp/pip_build_lowend/soundscrape/setup.py) egg_info for package soundscrape
    Downloading/unpacking https://bitbucket.org/lazka/mutagen/get/default.tar.gz
      Running setup.py (path:/tmp/pip-myuolr-build/setup.py) egg_info for package from https://bitbucket.org/lazka/mutagen/get/default.tar.gz
        /usr/local/lib/python2.7/dist-packages/setuptools-19.2-py2.7.egg/setuptools/dist.py:293: UserWarning: The version specified ('1.31.-1') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
    
      Requirement already satisfied (use --upgrade to upgrade): mutagen==1.31.-1 from https://bitbucket.org/lazka/mutagen/get/default.tar.gz in /usr/local/lib/python2.7/dist-packages
    Cleaning up...
    error: error in 'egg_base' option: 'pip-egg-info' does not exist or is not a directory
    Complete output from command python setup.py egg_info:
    Downloading/unpacking https://bitbucket.org/lazka/mutagen/get/default.tar.gz

  Running setup.py (path:/tmp/pip-myuolr-build/setup.py) egg_info for package from https://bitbucket.org/lazka/mutagen/get/default.tar.gz

    /usr/local/lib/python2.7/dist-packages/setuptools-19.2-py2.7.egg/setuptools/dist.py:293: UserWarning: The version specified ('1.31.-1') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.

  Requirement already satisfied (use --upgrade to upgrade): mutagen==1.31.-1 from https://bitbucket.org/lazka/mutagen/get/default.tar.gz in /usr/local/lib/python2.7/dist-packages

Cleaning up...

running egg_info

error: error in 'egg_base' option: 'pip-egg-info' does not exist or is not a directory

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip_build_lowend/soundscrape

.....................................................................................................

The fix is simply: 

*sudo pip install -U pip*

and then reinstall SoundScrape using pip!

s/o Miserlou I <3 SoundScrape!

**Comment by lowend** [comment]
fyi: this wasn't really an issue with SoundScrape, I just have had this happen multiple times on reinstalls and I wanted to spread the word.

**Comment by Miserlou** [comment]
Hm. Not sure what to make of this issue. 

Can you tell me a bit more about the system you're running this on?

Takk!
R

**Comment by walkerrandolphsmith** [comment]
`sudo -H pip install soundscrape` 

```
...
error in soundscrape setup command: Invalid environment marker: python_version < "3.0"
```

Any ideas?

**Comment by Miserlou** [comment]
Should be fixed in 0.23.6..

**Comment by ValidAQ** [comment]
There might be something related still going on? Getting this while trying to install (Win 8.1 x86, Python 3.4, pip 8.0.2):
```
pip install soundscrape
Collecting soundscrape
  Downloading soundscrape-0.23.7.tar.gz
No files/directories in C:\Users\admin\AppData\Local\Temp\pip-build-2tvyhnc4\soundscrape\pip-egg-info (from PKG-INFO)
```
Trying to figure out what this means, for now.
Should another issue be opened about it?
